File: app/dbmodels/NetworkPortVlan.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'NetworkPortVlan',
    'menu'          => ''
);
$table['fields'] = array(
    'id'              => DBModels::type('increments',
                                        array('visible' => false)),
    'network_port_id' => DBModels::type('integer'),
    'vlan_id'         => DBModels::type('integer'),
    'tagged'          => DBModels::type('boolean'),

);


$table['relationships'] = array(
    'vlan'     => array(
        'type'  => 'belongsTo',
        'item'  => 'Vlan'),
    'network_port' => array(
        'type'  => 'belongsTo',
        'item'  => 'NetworkPort'),
);

include "_commonentitylink.php";
include "_commonrecursiveentitylink.php";

File: fusionglpi/database/networkport_vlan.py
from fusionglpi import db


class NetworkportVlan(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    networkport_id = db.Column(db.Integer, db.ForeignKey('networkport.id'))
    vlan_id = db.Column(db.Integer, db.ForeignKey('vlan.id'))
    tagged = db.Column(db.Boolean, nullable=False, server_default="0")
